examples/main.tf

```diff
@@ -542,7 +542,7 @@ resource "instaclustr_cluster" "example-cadenceav-kafka" {
 
   bundle {
     bundle = "KAFKA"
-    version = "3.0.0"
+    version = "3.1.1"
```

Comment:
There's no need to change all the versions to 3.1.1. Especially for Cadence, if we want to change this, we'd better to test it whether this version can work well with Cadence.

Reply:
@vutdv-ic  @YuWang24  is right Cadence has a pinned version of Kafka (3.0.0) which is a hard dependency on the Kafka version. Don't change this unless Cadence team wants to change it.

Reply:
right, updated